src/main/java/edu/cornell/kfs/module/purap/document/validation/impl/CuCreditMemoDocumentPreRules.java

```diff
@@ -33,11 +36,13 @@ public boolean doPrompts(Document document){
         // KFSUPGRADE-779
         preRulesOK &= checkWireTransferTabState((VendorCreditMemoDocument) document);
 
+        preRulesOK &= validateCheckStubLength((VendorCreditMemoDocument) document);
+
         AccountsPayableDocument accountsPayableDocument = (AccountsPayableDocument) document;
 
         // Check if the total does not match the submitted credit if the document hasn't been completed.
         if (!SpringContext.getBean(PurapService.class).isFullDocumentEntryCompleted(accountsPayableDocument)) {
-            preRulesOK = confirmInvoiceNoMatchOverride(accountsPayableDocument);
+            preRulesOK &= confirmInvoiceNoMatchOverride(accountsPayableDocument);

```

Comment:
There are a few pre-rule tests above this one that affect the same variable. Without the `&=`, a successful outcome from this test could cause the document action to proceed, even if one of the prior tests failed.

For some reason, I think this whole `doPrompts()` method is a complete override that doesn't invoke the superclass variant, even though it includes most or all of the superclass's coding. It would be good to refactor the whole method at some point so that it invokes the superclass one (especially since this particular line might be in the superclass), but that would probably be beyond the scope of this PR.